### ISSUE TITLE
Unable to install qiskit on anaconda

### ISSUE THREAD

**Issue by fromillo** [Qiskit/qiskit-metapackage#658]
### Informations

- **Qiskit version: latest from pip**:
- **Python version 3.8.0**:
- **Operating system win10 with latest updates**:

### What is the current behavior?
I'm unable to install it in Anaconda (latest version, downloaded today)

### Steps to reproduce the problem
pip install qiskit

### What is the expected behavior?
i've this:
```
(qu1) C:\Users\Enrico>pip install qiskit
Collecting qiskit
  Using cached https://files.pythonhosted.org/packages/9e/2d/4c09bc1292a3be92f8c003b8491215ac9a0d81b014384e95a659a58ddf57/qiskit-0.13.0.tar.gz
Collecting qiskit-terra==0.10.0
  Using cached https://files.pythonhosted.org/packages/4e/86/180961c6ae2fecf521c5ea4246a3a3f52e9cbfed022c537733cc5203c228/qiskit-terra-0.10.0.tar.gz
Collecting qiskit-aer==0.3.2
  Downloading https://files.pythonhosted.org/packages/a8/46/11edbd99d7b732e63a542623109f394f17d9ad4051ddeb1e2dd9f36d1aac/qiskit-aer-0.3.2.tar.gz (6.4MB)
     |████████████████████████████████| 6.4MB 1.3MB/s
Collecting qiskit-ibmq-provider==0.3.3
  Downloading https://files.pythonhosted.org/packages/cb/6a/ae5f648a565492dd6a39d2472e8940434f2df36c9c4422a39bc9535bfe01/qiskit-ibmq-provider-0.3.3.tar.gz (57kB)
     |████████████████████████████████| 61kB 2.0MB/s
Collecting qiskit-ignis==0.2.0
  Downloading https://files.pythonhosted.org/packages/78/f2/f4f89c65d2c13f473092f7e800519836611ddeeaab8d4591a70555b26a37/qiskit_ignis-0.2.0-py3-none-any.whl (100kB)
     |████████████████████████████████| 102kB 6.4MB/s
Collecting qiskit-aqua==0.6.1
  Downloading https://files.pythonhosted.org/packages/47/2b/632c195974804d17dca01554caa013dd0c3023f4e4e6b91213c3381ce514/qiskit_aqua-0.6.1-py3-none-any.whl (979kB)
     |████████████████████████████████| 983kB 3.2MB/s
Collecting jsonschema>=2.6
  Downloading https://files.pythonhosted.org/packages/ce/6c/888d7c3c1fce3974c88a01a6bc553528c99d3586e098eee23e8383dd11c3/jsonschema-3.1.1-py2.py3-none-any.whl (56kB)
     |████████████████████████████████| 61kB 3.8MB/s
Collecting marshmallow<4,>=3
  Downloading https://files.pythonhosted.org/packages/78/8c/aa99cd72e69ce14c754a4df752a57faffbd698b14a6fda598a3950273e99/marshmallow-3.2.1-py2.py3-none-any.whl (45kB)
     |████████████████████████████████| 51kB 3.4MB/s
Collecting marshmallow_polyfield<6,>=5.7
  Downloading https://files.pythonhosted.org/packages/d7/6c/416e7e529d03362bb57bb4be16a0bb1ada6325b12adf3e31fec4c10ef1b1/marshmallow-polyfield-5.7.tar.gz
Collecting networkx>=2.2
  Using cached https://files.pythonhosted.org/packages/41/8f/dd6a8e85946def36e4f2c69c84219af0fa5e832b018c970e92f2ad337e45/networkx-2.4-py3-none-any.whl
Collecting numpy>=1.13
  Using cached https://files.pythonhosted.org/packages/90/4e/98818cb208f32833f628d7f7e9dd9ce36cdc34d199ccae0ab37ed6a13b85/numpy-1.17.3-cp38-cp38-win_amd64.whl
Collecting ply>=3.10
  Downloading https://files.pythonhosted.org/packages/a3/58/35da89ee790598a0700ea49b2a66594140f44dec458c07e8e3d4979137fc/ply-3.11-py2.py3-none-any.whl (49kB)
     |████████████████████████████████| 51kB 1.7MB/s
Collecting psutil>=5
  Downloading https://files.pythonhosted.org/packages/a7/7f/0761489b5467af4e97ae3c5a25f24f8662d69a25692d85490c8ea5d52e45/psutil-5.6.3-cp38-cp38-win_amd64.whl (238kB)
     |████████████████████████████████| 245kB 6.8MB/s
Collecting scipy>=1.0
  Downloading https://files.pythonhosted.org/packages/ee/5b/5afcd1c46f97b3c2ac3489dbc95d6ca28eacf8e3634e51f495da68d97f0f/scipy-1.3.1.tar.gz (23.6MB)
     |████████████████████████████████| 23.6MB 6.8MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\Enrico\Anaconda3\envs\qu1\python.exe' 'C:\Users\Enrico\Anaconda3\envs\qu1\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\Enrico\AppData\Local\Temp\tmpeda_bjma'
         cwd: C:\Users\Enrico\AppData\Local\Temp\pip-install-0srgf6fw\scipy
    Complete output (216 lines):
    lapack_opt_info:
    lapack_mkl_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries mkl_rt not found in ['C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\lib', 'C:\\', 'C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\libs', 'C:\\Users\\Enrico\\Anaconda3\\Library\\lib']
      NOT AVAILABLE

    openblas_lapack_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries openblas not found in ['C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\lib', 'C:\\', 'C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\libs', 'C:\\Users\\Enrico\\Anaconda3\\Library\\lib']
    get_default_fcompiler: matching types: '['gnu', 'intelv', 'absoft', 'compaqv', 'intelev', 'gnu95', 'g95', 'intelvem', 'intelem', 'flang']'
    customize GnuFCompiler
    Could not locate executable g77
    Could not locate executable f77
    customize IntelVisualFCompiler
    Could not locate executable ifort
    Could not locate executable ifl
    customize AbsoftFCompiler
    Could not locate executable f90
    customize CompaqVisualFCompiler
    Could not locate executable DF
    customize IntelItaniumVisualFCompiler
    Could not locate executable efl
    customize Gnu95FCompiler
    Could not locate executable gfortran
    Could not locate executable f95
    customize G95FCompiler
    Could not locate executable g95
    customize IntelEM64VisualFCompiler
    customize IntelEM64TFCompiler
    Could not locate executable efort
    Could not locate executable efc
    customize PGroupFlangCompiler
    Could not locate executable flang
    don't know how to compile Fortran code on platform 'nt'
      NOT AVAILABLE

    openblas_clapack_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries openblas,lapack not found in ['C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\lib', 'C:\\', 'C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\libs', 'C:\\Users\\Enrico\\Anaconda3\\Library\\lib']
      NOT AVAILABLE

    atlas_3_10_threads_info:
    Setting PTATLAS=ATLAS
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries tatlas,tatlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries tatlas,tatlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries tatlas,tatlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries tatlas,tatlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    <class 'numpy.distutils.system_info.atlas_3_10_threads_info'>
      NOT AVAILABLE

    atlas_3_10_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries satlas,satlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries satlas,satlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries satlas,satlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries satlas,satlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    <class 'numpy.distutils.system_info.atlas_3_10_info'>
      NOT AVAILABLE

    atlas_threads_info:
    Setting PTATLAS=ATLAS
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries ptf77blas,ptcblas,atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries ptf77blas,ptcblas,atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries ptf77blas,ptcblas,atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries ptf77blas,ptcblas,atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    <class 'numpy.distutils.system_info.atlas_threads_info'>
      NOT AVAILABLE

    atlas_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries f77blas,cblas,atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries f77blas,cblas,atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries f77blas,cblas,atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\envs\qu1\libs
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries f77blas,cblas,atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack_atlas not found in C:\Users\Enrico\Anaconda3\Library\lib
    <class 'numpy.distutils.system_info.atlas_info'>
      NOT AVAILABLE

    lapack_info:
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
      libraries lapack not found in ['C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\lib', 'C:\\', 'C:\\Users\\Enrico\\Anaconda3\\envs\\qu1\\libs', 'C:\\Users\\Enrico\\Anaconda3\\Library\\lib']
      NOT AVAILABLE

    lapack_src_info:
      NOT AVAILABLE

      NOT AVAILABLE

    setup.py:386: UserWarning: Unrecognized setuptools command ('dist_info --egg-base C:\Users\Enrico\AppData\Local\Temp\pip-modern-metadata-5n159c6v'), proceeding with generating Cython sources and expanding templates
      warnings.warn("Unrecognized setuptools command ('{}'), proceeding with "
    Running from scipy source directory.
    C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\numpy\distutils\system_info.py:624: UserWarning:
        Atlas (http://math-atlas.sourceforge.net/) libraries not found.
        Directories to search for the libraries can be specified in the
        numpy/distutils/site.cfg file (section [atlas]) or by setting
        the ATLAS environment variable.
      self.calc_info()
    C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\numpy\distutils\system_info.py:624: UserWarning:
        Lapack (http://www.netlib.org/lapack/) libraries not found.
        Directories to search for the libraries can be specified in the
        numpy/distutils/site.cfg file (section [lapack]) or by setting
        the LAPACK environment variable.
      self.calc_info()
    C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\numpy\distutils\system_info.py:624: UserWarning:
        Lapack (http://www.netlib.org/lapack/) sources not found.
        Directories to search for the sources can be specified in the
        numpy/distutils/site.cfg file (section [lapack_src]) or by setting
        the LAPACK_SRC environment variable.
      self.calc_info()
    Traceback (most recent call last):
      File "C:\Users\Enrico\Anaconda3\envs\qu1\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 257, in <module>
        main()
      File "C:\Users\Enrico\Anaconda3\envs\qu1\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 240, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "C:\Users\Enrico\Anaconda3\envs\qu1\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 110, in prepare_metadata_for_build_wheel
        return hook(metadata_directory, config_settings)
      File "C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\setuptools\build_meta.py", line 156, in prepare_metadata_for_build_wheel
        self.run_setup()
      File "C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\setuptools\build_meta.py", line 236, in run_setup
        super(_BuildMetaLegacyBackend,
      File "C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\setuptools\build_meta.py", line 142, in run_setup
        exec(compile(code, __file__, 'exec'), locals())
      File "setup.py", line 505, in <module>
        setup_package()
      File "setup.py", line 501, in setup_package
        setup(**metadata)
      File "C:\Users\Enrico\AppData\Local\Temp\pip-build-env-3vozxafd\overlay\Lib\site-packages\numpy\distutils\core.py", line 135, in setup
        config = configuration()
      File "setup.py", line 403, in configuration
        raise NotFoundError(msg)
    numpy.distutils.system_info.NotFoundError: No lapack/blas resources found.
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\Enrico\Anaconda3\envs\qu1\python.exe' 'C:\Users\Enrico\Anaconda3\envs\qu1\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\Enrico\AppData\Local\Temp\tmpeda_bjma' Check the logs for full command output.
```

### Suggested solutions
i've no idea :(

**Comment by mtreinish** [comment]
Qiskit does not support python 3.8.0 yet. This is mostly because it has dependencies on libraries that do not have published wheels on python 3.8.0 yet. This is what you're hitting scipy has not published a release with a precompiled binary on 3.8 yet and when you go to install it in pip is trying to install scipy from source, which fails because you do not have a lapack/blas library source installed. There is an open PR on qiskit-terra to track adding python 3.8 support, but it is blocked on upstream dependencies like scipy: https://github.com/Qiskit/qiskit-terra/pull/3268 

Until that merges I'd just recommend building a new anaconda env with python 3.7. This should be straightforward to do with conda create.

**Comment by fromillo** [comment]
Thank you for your suggestions. I followed the instructions on the site, but i didn't saw that 3.8 isn't supported yet.

I switched to 3.7.4 and everything seems to work.

Thank you very much
